cms/static/js/views/import.js

```diff
@@ -108,6 +108,9 @@ define(
                     $(elem).find('p.copy').show();
                     updateCog($(elem), false);
                 });
+                all.find('.fa-check-square-o'). // Replace checkmark with unchecked box
```

Comment:
Why does this need to be done in the JS as opposed to where the HTML is constructed?